src/repository/base.repository.js

```diff
@@ -139,7 +150,31 @@ class BaseRepository {
    * @memberof BaseRepository
    */
   static findOne(model, options) {
-    return model.findByPk(options);
```

Comment:
I think this method already exist on the project and there is probably a reason why `findByPk` was used. Please return this to the way it was. Moreso, I think this is a repetition on the method on line `49`